### PR TITLE
added pointers to new polymerfire component

### PR DESCRIPTION
**Pull request by tjsavage**

### CODE REVIEW

**Comment by tjsavage** [comment]
@ebidel WDYT? I'd like to get the Polymer Elements Catalog entries updated as soon as we can.

**Comment by cdata** [comment]
LGTM 👍

File: firebase-collection.html [hunk 1]
```diff
@@ -12,6 +12,10 @@
 <link rel="import" href="firebase-query-behavior.html">
 
 <!--
+**Note: This element is for the older Firebase 2 API**
+For the latest official Firebase 3.0-compatible component from the Firebase team,
+see the [polymerfire](https://github.com/firebase/polymerfire) component.
+
```
Comment (chuckh) [inline firebase-collection.html]: You also might want to note that `firebase-collection` is replaced with `firebase-query` and link to https://github.com/firebase/polymerfire#firebase-query

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-firebase-element
+## Note: This element is for the older Firebase 2.0 API.
+For the latest official Firebase 3.0-compatible component from the Firebase team,
```
Comment (ebidel) [inline README.md]: 2.0

**Comment by ebidel** [comment]
LGTM. Want to add @chuckh 's comment?

**Comment by chuckh** [comment]
LGTM 👍

**Comment by ebidel** [comment]
LGTM Travis looks like it's having issues @cdata @tjsavage do you know what's up with it?

**Comment by cdata** [comment]
@ebidel It's possible that the travis config for this repo is out of date. I'm not sure if Tedium runs on it or not.

**Pull request merged**